dev/tests/functional/tests/app/Magento/Checkout/Test/Constraint/AssertCartPerCustomer.php

```diff
@@ -53,7 +53,10 @@ public function processAssert(
                     ['customer' => $customer]
                 )->run();
                 \PHPUnit_Framework_Assert::assertEquals(
-                    sprintf(self::WELCOME_MESSAGE, $customer->getFirstname()),
+                    sprintf(
+                        self::WELCOME_MESSAGE,
+                        $customer->getFirstname() . ' ' . $customer->getLastname()
```

Comment:
This assertion may lead to false negative result in a future as full name has more complex logic then just first and last name (see `\Magento\Customer\Helper\View::getCustomerName`)

Reply:
@vkublytskyi You're right, but the fixtures are autogenerated - such complex logic is just not expected there.